test/replcompletions.jl

```diff
@@ -457,7 +457,6 @@ try
     end
 finally
     @test pop!(LOAD_PATH) == path
-    rm(path, recursive=true)
```

Comment:
the parent dirs get created. and what does this have to do with the rest of this pr?